src/index/VocabularyGeneratorImpl.h

```diff
@@ -215,37 +215,55 @@ void VocabularyMerger::doActualWrite(
   }
 }
 
-// ______________________________________________________________________________________________
-template <class Pred>
-void writePartialIdMapToBinaryFileForMerging(std::shared_ptr<const Index::ItemMap> map,
-                                             const string& fileName, Pred pred) {
-  LOG(INFO) << "Creating partial vocabulary from set ...\n";
-  std::vector<std::pair<string, Id>> els;
-  els.reserve(map->size());
-  els.insert(begin(els), begin(*map), end(*map));
-  LOG(INFO) << "... sorting ...\n";
-
-  auto sort = [&els](const auto& pred) {
-    if constexpr (USE_PARALLEL_SORT) {
-      __gnu_parallel::sort(begin(els), end(els), pred,
-                           __gnu_parallel::parallel_tag(NUM_SORT_THREADS));
-    } else {
-      std::sort(begin(els), end(els), pred);
+// ____________________________________________________________________________________________________________
+absl::flat_hash_map<Id, Id> createInternalMapping(std::vector<std::pair<string, Id>>* elsPtr) {
```

Comment:
I'd like to stick to just one hash map implementation in normal QLever code (i.e. in libraries it's ok). So I think we should make `util::HashMap` use `absl::flat_hash_map` I think I once refactored it so that this shouldn't be too hard. What do you think?

Reply:
For this PR I removed `absl` again, since the parallelism takes care of my current performance issues and we can do the change in a clean separate PR.